public/app/features/alerting/unified/components/rules/state-history/LogTimelineViewer.tsx

```diff
@@ -20,7 +19,6 @@ const replaceVariables: InterpolateFunction = (v) => v;
 
 export const LogTimelineViewer = React.memo(({ frames, timeRange, onPointerMove = noop }: LogTimelineViewerProps) => {
   const theme = useTheme2();
-  const { setupCursorTracking } = useCursorTimelinePosition(onPointerMove);
```

Comment:
Small nit: can we remove `onPointerMove` from the interface / props here? It no longer seems to be used :)

Reply:
:+1:

Reply:
also had to remove it from LokiStateHistory in https://github.com/grafana/grafana/pull/84420/commits/d8dbefb6dc1aeb5d12ae7529b97c425ee048e605